openslides_backend/services/media/adapter.py

```diff
@@ -12,16 +13,32 @@ class MediaServiceAdapter(MediaService):
 
     def __init__(self, media_url: str, logging: LoggingModule) -> None:
         self.logger = logging.getLogger(__name__)
-        self.media_url = media_url + "/upload"
+        self.media_url = media_url + "/"
 
-    def upload(self, file: str, id: int, mimetype: str) -> None:
+    def _upload(
+        self, file: str, id: int, mimetype: str, subpath: str
+    ) -> Union[str, None]:
+        url = self.media_url + subpath
         payload = {"file": file, "id": id, "mimetype": mimetype}
         self.logger.debug("Starting upload of file")
         try:
-            response = requests.post(self.media_url, json=payload)
+            response = requests.post(url, json=payload)
         except requests.exceptions.ConnectionError:
-            raise ServiceException("Connect to mediaservice failed.")
+            msg = "Connect to mediaservice failed."
+            self.logger.debug("Upload of file: " + msg)
+            return msg
 
         if response.status_code != 200:
-            raise ServiceException(f"Mediaservice Error: {str(response.content)}")
+            msg = f"Mediaservice Error: {str(response.content)}"
+            self.logger.debug("Upload of file: " + msg)
+            return msg
         self.logger.debug("File successfully uploaded to the media service")
+        return None
```

Comment:
leas do not use return values to propagate error, but use exceptions. Create a `MediafileUploadException` or so.

Reply:
Done